### ISSUE TITLE
[TESTS] Need tests for bookmarks

### ISSUE THREAD

**Issue by lucasace** [pesos/browser-history#118]
Currently, tests test only histories
Tests for bookmarks need to be built as well

# Required Tests

## Browsers
  | Platform | Linux | Mac | Windows
-- | -- | -- | -- | --
Browser |   |   |   |  
Firefox |   |  :x: | :x:  | ❌   
Chrome |   |  ❌  | ❌    |:x:  
Chromium |   |:x:   | ❌   | ❌  
Safari |   |  🔲 |  🔲  | 🔲 
Edge |   |🔲 | ❌   | ❌   
Opera |   |  ❌  | ❌   | ❌  
Opera GX |   |  🔲 |   🔲|  ❌ 
Brave  |  | ❌ | ❌ |:x:|

**Comment by Samyak2** [comment]
Just to confirm, :black_square_button: means the browser isn't supported for bookmarks, right?

**Comment by lucasace** [comment]
Yup, but not entirely it also means that the Browser is not available in that OS, Ex Safari in Linux
